iac-extensions/docker/src/test/java/org/sonar/iac/docker/TestUtils.java

```diff
@@ -45,4 +46,22 @@ public static void assertKeyValuePair(NewKeyValuePair keyValuePair, String expec
       assertThat(argValue(keyValuePair.value())).isEqualTo(expectedValue);
     }
   }
+
+  public static void assertFrom(FromInstruction from, String expectedName, @Nullable String expectedFlagName, @Nullable String expectedFlagValue, @Nullable String expectedAlias) {
```

Comment:
Minor. There is a better way of implementing assertions in a more fluent way, see: `TextRangeAssert` and https://assertj.github.io/doc/#assertj-core-custom-assertions

then the code:

```
assertFrom(from, "foobar", null, null, null);
```

can be replaced by:

```
assertThat(from).hasName("foobar");
```

The current form is not bad, consider refactor it or not - it's up to you :-)

Reply:
Interesting approach ! I tried both, while I prefer the second usage over the first one, the implementation is quite bigger (x2), which is a bit of an overkill for an element that is not much used. I discussed with Nils about it, he also would like to keep it simple as this element is not used at many place and is not expose in a shared library.
But I will definitely keep this approach in mind for an other appropriate use case.